### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by asood123**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/stylelint ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

**Comment by hudochenkov** [comment]
Duplicate of https://github.com/stylelint/stylelint/pull/2707?

**Comment by asood123** [comment]
Ah yes, missed that somehow.

**Comment by ntwb** [comment]
Can you restore the branch here @asood123 ?

**Comment by asood123** [comment]
@ntwb branch still exists. https://github.com/stylelint/stylelint/compare/master...opencollective:opencollective?expand=1. It's already linked to this PR? Is there something special I need to do for you to be able to merge it?

**Comment by ntwb** [comment]
@asood123 How do I edit the "Become a sponsor"  and "Become a backer" and create other events?

When I try I get an error: `You need to be logged in as a core contributor of this collective to be able to create an event.`, can you set that up for me please

https://opencollective.com/stylelint

File: package.json [hunk 1]
```diff
@@ -116,7 +117,8 @@
     "prettier:fix": "prettier '**/*.js' --write",
     "release": "npmpub",
     "test": "jest --coverage",
-    "watch": "jest --watch"
+    "watch": "jest --watch",
+    "postinstall": "opencollective postinstall"
```
Comment (ntwb) [inline package.json]: FYI: This installs and runs an _opencollective_ CLI script and upon running `npm install` will display a prompt to invite the user to donate after installing your package.

**Comment by ntwb** [comment]
@stylelint/core Open Collective is now activated for stylelint https://opencollective.com/stylelint

I've started a discussion for the team at https://github.com/orgs/stylelint/teams/core

**Comment by hudochenkov** [comment]
Not a nice error after `npm install`:

image

**Comment by xdamman** [comment]
@hudochenkov thanks for reporting. This has been fixed (it was on our side)

**Comment by astorije** [comment]
Hey there! This post-install message is now showing (or rather SHOWING) after installing any project that has `stylelint` in their dependencies.
It is very noisy, would you mind removing it please? I can't imagine if Webpack, Babel, eslint, etc. start adding this :(

screen shot 2018-02-19 at 12 32 05

**Comment by xdamman** [comment]
You can also do a more subtle native one liner like Preact did: https://github.com/developit/preact/blob/master/package.json#L39

**Pull request merged**